src/components/ContentWidgets/ImagesInRow/ImagesInRow.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { ContentSingularData } from '../../_data/ContentSingularData';
 import './ImagesInRow.css';
 
-type linkCaptionPair = {
+type imgRowItem = {
     imgLink: string
     imgCaption: string
     extLink: string
```

Comment:
`extLink` is not a clear variable name. It's best just to be super explicit with these.